api/validators.py

```diff
@@ -105,10 +103,10 @@ def g(payload, method):
             try:
                 _validate_json(payload, _schema, resolver_input)
             except jsonschema.ValidationError as e:
-                handler.abort(400, str(e))
+                raise webapp2.exc.HTTPBadRequest(str(e))
```

Comment:
yeah probably the best thing to do is to catch the ValidationError in the handler calling the method